src/Surfnet/Stepup/Configuration/Api/InstitutionConfiguration.php

```diff
@@ -25,15 +25,36 @@
 use Surfnet\Stepup\Configuration\Value\Location;
 use Surfnet\Stepup\Configuration\Value\RaLocationId;
 use Surfnet\Stepup\Configuration\Value\RaLocationName;
+use Surfnet\Stepup\Configuration\Value\ShowRaaContactInformationOption;
+use Surfnet\Stepup\Configuration\Value\UseRaLocationsOption;
 
 interface InstitutionConfiguration extends AggregateRoot
 {
     /**
      * @param InstitutionConfigurationId $institutionConfigurationId
      * @param Institution $institution
+     * @param UseRaLocationsOption $useRaLocationsOption
+     * @param ShowRaaContactInformationOption $showRaaContactInformationOption
      * @return InstitutionConfiguration
      */
-    public static function create(InstitutionConfigurationId $institutionConfigurationId, Institution $institution);
+    public static function create(
+        InstitutionConfigurationId $institutionConfigurationId,
+        Institution $institution,
+        UseRaLocationsOption $useRaLocationsOption,
+        ShowRaaContactInformationOption $showRaaContactInformationOption
+    );
+
+    /**
+     * @param UseRaLocationsOption $useRaLocationsOption
+     * @return void
+     */
+    public function changeUseRaLocationsOption(UseRaLocationsOption $useRaLocationsOption);
+
+    /**
+     * @param ShowRaaContactInformationOption $raaContactInformationOption
+     * @return void
+     */
+    public function changeShowRaaContactInformationOption(ShowRaaContactInformationOption $raaContactInformationOption);
```

Comment:
`set` vs `change`? Seems change would imply a change, regardless of whether or not there actually is a change... set is a bit of a nono, however, you're setting an option, something that is actually quite normal to do. Anyways, might want to give it some thought. Not saying it has to change, just that there might be a better alternative...